storm-core/src/clj/org/apache/storm/testing.clj

```diff
@@ -288,11 +289,11 @@
 (defmacro with-inprocess-zookeeper
   [port-sym & body]
   `(with-local-tmp [tmp#]
-                   (let [[~port-sym zks#] (zk/mk-inprocess-zookeeper tmp#)]
+                   (let [[~port-sym zks#] (Zookeeper/mkInprocessZookeeper tmp# nil)]
```

Comment:
This is not a drop in replacement because mkInprocessZookeeper is returning just zks# not ~port-sym.  You might be able to make this work by changing it to.

```
   (let [zks# (Zookeeper/mkInprocessZookeeper tmp# nil)
          ~port-sym (.getLocalPort zks#)]
```